src/components/search/curated/[slug]/index.tsx

```diff
@@ -156,94 +261,125 @@ const CuratedTopic: React.FC<CuratedTopicProps> = ({topic, topicData}) => {
-                              location={location}
-                              completedCoursesIds={completedCoursesIds}
-                            />
-                          )
-                        case 3:
-                          return i === 0 ? (
-                            <HorizontalResourceCard
-                              className="col-span-2"
-                              key={resource.title}
-                              resource={resource}
-                              location={location}
-                              completedCoursesIds={completedCoursesIds}
-                            />
-                          ) : (
-                            <VerticalResourceCard
-                              key={resource.title}
-                              resource={resource}
-                              location={location}
-                              completedCoursesIds={completedCoursesIds}
-                            />
-                          )
-                        case 6:
-                          return i === 0 || i === 1 ? (
-                            <HorizontalResourceCard
-                              className="col-span-2"
-                              key={resource.title}
-                              resource={resource}
-                              location={location}
-                              completedCoursesIds={completedCoursesIds}
-                            />
-                          ) : (
-                            <VerticalResourceCard
-                              key={resource.title}
-                              resource={resource}
-                              location={location}
-                              completedCoursesIds={completedCoursesIds}
-                            />
-                          )
-                        case 7:
-                          return i === 0 ? (
-                            <HorizontalResourceCard
-                              className="col-span-2"
-                              key={resource.title}
-                              resource={resource}
-                              location={location}
-                              completedCoursesIds={completedCoursesIds}
-                            />
-                          ) : (
-                            <VerticalResourceCard
-                              key={resource.title}
-                              resource={resource}
-                              location={location}
-                              completedCoursesIds={completedCoursesIds}
-                            />
-                          )
-                        default:
+                      <div className="grid md:grid-cols-4 gap-4 ">
+                        <div
+                          className={`col-span-2  ${cx({
+                            'md:order-last': i % 2 !== 0,
+                          })}`}
+                        >
```

Comment:
flipping the text to the opposite column because having text stacked on the left side looked a little off. Maybe we want to do this differently?

This is what it looks like without the flipped text:
![In-Depth Redux Tutorials for 2023  egghead io](https://user-images.githubusercontent.com/6188161/233501457-34ce24c9-e69a-40ed-9f48-a84c5b01ed77.png)